tests/snuba/api/endpoints/test_organization_events_mep.py

```diff
@@ -445,6 +445,7 @@ def test_performance_homepage_query(self):
             assert field_meta["failure_count()"] == "integer"
 
     def test_user_misery_and_team_key_sort(self):
+        self.features["organizations:use-metrics-layer"] = True
```

Comment:
Is this intentional? use-metrics-layer is enabled by the `OrganizationEventsMetricsEnhancedPerformanceEndpointTestWithMetricLayer` tests

Reply:
This is my bad, i tend to hardcode it for doing some tests quicker, will remove it.